packages/internal-test-helpers/lib/module-for.js

```diff
@@ -29,7 +33,9 @@ export function setupTestClass(hooks, TestClass, ...mixins) {
       promises.push(instance.afterEach());
     }
 
-    return all(promises);
+    return all(promises).then(() => {
```

Comment:
Should probably be `finally` (we want to unset even if rejected)

Reply:
agreed, fixed